app/assets/stylesheets/sbstyles.scss

```diff
@@ -0,0 +1,26 @@
+body {
+  background-image: image_url("background1.jpg");
```

Comment:
Name of function `image_url` should be written in all lowercase letters with hyphens instead of underscores